src/analytics/controllers/dashboard.py

```diff
@@ -382,23 +372,22 @@ def generate_customised_chart_data():
                         datasets.append(dataset)
                     measurement_units = '(µg/m3)'
                     if pollutant == 'NO2':
-                        measurement_units = ' Concentration'
-                    chart_label = pollutant + measurement_units
-
-                    custom_chat_data.append({'start_date': start_date, 'end_date': end_date, 'division': division,
-                                             'parish': parish, 'frequency': frequency, 'pollutant': pollutant,
-                                             'location_code': location_code, 'chart_type': chart_type, 'chart_data': device_results,
-                                             'datasets': datasets, 'custom_chart_title': custom_chart_title, 'chart_label': chart_label})
-
-                locations_devices.append(devices)
-
-        return jsonify({'results': custom_chat_data, 'datasets': datasets, 'custom_chart_title': custom_chart_title})
-
-        # else:
-        # return jsonify({'inputs': json_data,'errors': errors})
-
-
-@dashboard_bp.route('/api/v1/device/custom/chart/ANQ16PZJ', methods=['GET'])
+                        measurement_units = ' Concentration'    
```

Comment:
just wondering, is this device code (ANQ16PZJ) supposed to be a variable...are we are targeting one specific device for this endpoint? @sserurich

Reply:
@Baalmart , that's fine for now,  I think it can be moved and adjusted when the defaults implementation is in place, I kinder of not even utelise that endpoint.

Reply:
@Baalmart , I have removed the endpoint/method.

Reply:
Cool.